camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/mapred/EtlInputFormat.java

```diff
@@ -416,8 +405,10 @@ public int compare(CamusRequest r1, CamusRequest r2) {
       if (request.getEarliestOffset() > request.getOffset() || request.getOffset() > request.getLastOffset()) {
         if (request.getEarliestOffset() > request.getOffset()) {
           log.error("The earliest offset was found to be more than the current offset: " + request);
+          EmailClient.sendEmail("The earliest offset was found to be more than the current offset: " + request);
```

Comment:
I don't know specific use cases, so this might be wrong. Even if it is not necessarily an error situation, shouldn't we still send an email as a warning? One of these messages when a new topic is added is ok, but if they are being received often, that seems like a sign of an actual issue with the service, and the emails might be the first notice.